### ISSUE TITLE
Focal Fossa (20.04) is unable to create images for CentOS7/Rocky8/RHEL8

### ISSUE THREAD

**Issue by mmartinez117** [canonical/packer-maas#209]
OS: Ubuntu 20.04
Packer Version: 1.10.1
Make file/configs are entirely stock, direct from the repo, I haven't changed a line.

Right now, whenever I run, either, `sudo make` or `sudo packer build .` it will go through the process of downloading the ISO for any of the above OS and will then proceed to hang on the 'waiting for shutdown...' for the entire hour of the timeout window and error out.

I have removed and reinstalled packer three times and the error persists.

output of `kvm-ok`
```
hybridos@normandy:~/packer-maas-main/rocky8$ kvm-ok
INFO: /dev/kvm exists
KVM acceleration can be used
```

Log file attached

[packer-log1355405694.txt](https://github.com/canonical/packer-maas/files/14300713/packer-log1355405694.txt)

**Comment by alexsander-souza** [comment]
Hi, can you run it again with `PACKER_LOG=1` set in the environment?

**Comment by mmartinez117** [comment]
> Hi, can you run it again with `PACKER_LOG=1` set in the environment?

I've got the logs from last week when I was initially attempting Rocky8.

Magically, the program seems to have figured out how to work over the weekend after I was able to get it to function on 22.04 using another iteration of same hardware at my home lab.

The log file it attached.
[rocky8LOG.txt](https://github.com/canonical/packer-maas/files/14346945/rocky8LOG.txt)
